test/templates/table/index.njk

```diff
@@ -281,11 +243,7 @@
                 label: "Header Column",
                 id: "col1",
                 context_menu: true,
-                menu: [
-                    { label: "Ascending" },
-                    { label: "Decensing" },
-                    { label: "Fix Column" }
-                ],
+                menu: [],
```

Comment:
We can probably just remove the menu option from the table macro because it's handled by popper.